spack/package.py

```diff
@@ -237,7 +237,15 @@ def setup_build_environment(self, env):
         env.set("COMP_SYS", comp_sys)
 
     def build(self, pkg, spec, prefix):
-        # Get source files for netCDF builds
+        # Get source files for netCDF4 builds
```

Comment:
Is it normal for spack files to contain URLs of other packages? @AlexanderRichert-NOAA do you know?

Reply:
No, or at least not in this way. If these are really needed, it should probably be done with what Spack calls 'resources': https://spack.readthedocs.io/en/latest/packaging_guide.html#resources-expanding-extra-tarballs

Reply:
I don't think they are needed. We can just handle the dependencies the usual way...

Reply:
Am I removing these "wget" lines for netcdf? I believe they were from previous PRs.

Reply:
Yes, remove them.